doc-content/shared-kie-docs/src/main/asciidoc/KieServer/controller-rest-api-templates-ref.adoc

```diff
@@ -0,0 +1,881 @@
+
+== [GET] /management/servers/{serverId}
+
+Returns information about a specified {KIE_SERVER} template.
+
+.Request parameters
+[cols="25%,45%,15%,15%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Type
+|Requirement
+
+|`serverId`
+|ID of the {KIE_SERVER} template to be retrieved
+|String
+|Required
+|===
+
+.Example server response (JSON)
+[source,json]
+----
+{
+  "server-id": "default-kieserver",
+  "server-name": "default-kieserver",
+  "container-specs": [
+    {
+      "container-id": "employeerostering_1.0.0-SNAPSHOT",
+      "container-name": "employeerostering",
+      "server-template-key": {
+        "server-id": "default-kieserver",
+        "server-name": "default-kieserver"
+      },
+      "release-id": {
+        "group-id": "employeerostering",
+        "artifact-id": "employeerostering",
+        "version": "1.0.0-SNAPSHOT"
+      },
+      "configuration": {
+        "RULE": {
+          "org.kie.server.controller.api.model.spec.RuleConfig": {
+            "pollInterval": null,
+            "scannerStatus": "STOPPED"
+          }
+        },
+        "PROCESS": {
+          "org.kie.server.controller.api.model.spec.ProcessConfig": {
+            "runtimeStrategy": "SINGLETON",
+            "kbase": "",
+            "ksession": "",
+            "mergeMode": "MERGE_COLLECTIONS"
+          }
+        }
+      },
+      "status": "STARTED"
+    },
+    {
+      "container-id": "mortgage-process_1.0.0-SNAPSHOT",
+      "container-name": "mortgage-process",
+      "server-template-key": {
+        "server-id": "default-kieserver",
+        "server-name": "default-kieserver"
+      },
+      "release-id": {
+        "group-id": "mortgage-process",
+        "artifact-id": "mortgage-process",
+        "version": "1.0.0-SNAPSHOT"
+      },
+      "configuration": {
+        "RULE": {
+          "org.kie.server.controller.api.model.spec.RuleConfig": {
+            "pollInterval": null,
+            "scannerStatus": "STOPPED"
+          }
+        },
+        "PROCESS": {
+          "org.kie.server.controller.api.model.spec.ProcessConfig": {
+            "runtimeStrategy": "PER_PROCESS_INSTANCE",
+            "kbase": "",
+            "ksession": "",
+            "mergeMode": "MERGE_COLLECTIONS"
+          }
+        }
+      },
+      "status": "STARTED"
+    }
+  ],
+  "server-config": {},
+  "server-instances": [
+    {
+      "server-instance-id": "default-kieserver-instance@localhost:8080",
+      "server-name": "default-kieserver-instance@localhost:8080",
+      "server-template-id": "default-kieserver",
+      "server-url": "http://localhost:8080/kie-server/services/rest/server"
+    }
+  ],
+  "capabilities": [
+    "RULE",
+    "PROCESS",
+    "PLANNING"
+  ]
+}
+----
+
+== [PUT] /management/servers/{serverId}
+
+Creates a new {KIE_SERVER} template with a specified ID.
+
+.Request parameters
+[cols="25%,45%,15%,15%", frame="all", options="header"]
+|===
+|Name
+|Description
+|Type
+|Requirement
+
+|`serverId`
```

Comment:
wondering if we should use _serverTemplateId_ instead, @sterobin @mswiderski wdyt?

Reply:
Yes, agreed! Great point. I've updated my doc content accordingly. Please do the same as you apply the doc changes to the Swagger page.